apps/addresses/models.py

```diff
@@ -16,19 +16,22 @@ class BasePostalAddress(AbstractGeolocation):
     This object is always referenced by a foreign key from another object
 
     Storing address as a separate model is a best practice
-    Other than that, there are none. Parsing postal addresses and (even human names, for that matter) are notoriously difficult (not even taking international addresses into consideration)
+    Other than that, there are none. Parsing postal addresses and (even human names, for that matter) are
+    notoriously difficult (not even taking international addresses into consideration)
 
     http://stackoverflow.com/questions/310540/best-practices-for-storing-postal-addresses-in-a-database-rdbms
 
-    The safest way would be to have a free-form field to enter whatever address, and then an algorithm that can be maintained over time that parses the address into structured fields.
+    The safest way would be to have a free-form field to enter whatever address, and then an algorithm that can
+    be maintained over time that parses the address into structured fields.
 
     Right now, go with the "good enough" approach
     """
     name = models.CharField(max_length=64, blank=True)
     street = models.CharField(max_length=128, blank=True)
+    neighborhood = models.CharField(max_length=64, blank=True)
     city = models.CharField(max_length=64, blank=True)
     state = models.CharField(max_length=2, blank=True)
-    zipcode = models.CharField(max_length=5, blank=True)
+    zipcode = models.CharField(max_length=16, blank=True)
```

Comment:
## REVIEWER NOTE:
Changed `max_length` to make it compatible with `zippy model`

Reply:
Somewhat leaning toward keeping it as is for now, and modifying it in the subclass